add changes

xaler5 opened the pull request:
Closing
https://github.com/OpenZeppelin/ethernaut/pull/526

howjmay commented:
Isn't it better to use the github feature `coauthored`? Otherwise it is quite sad the @OjusWiZard and I are not included in the commit history tho

xaler5 commented:
Thanks for pointing it out @howjmay ! My bad as I accidentally lost track of your commits, it was completely unintetional.

What I did to soft-fix it:
- Reverted the merge
- Rebased with master (it had changes after this PR)
- pushed it again in master

I've added a co-authored commit on top of the last push but it points to the rebasing with master, where there are other changes too.

I'm gonna keep that in mind for next iteration. Sorry for the inconvenience

howjmay commented:
Thank you so much!!
